kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableImpl.java

```diff
@@ -38,24 +38,26 @@ public static Table forPath(TableClient tableClient, String path)
         } catch (IOException io) {
             throw new RuntimeException(io);
         }
-        final Path dataPath = new Path(resolvedPath);
-        final Path logPath = new Path(dataPath, "_delta_log");
-
-        return new TableImpl(logPath, dataPath);
+        return new TableImpl(resolvedPath);
     }
 
-    private final Path logPath;
-    private final Path dataPath;
     private final SnapshotManager snapshotManager;
+    private final String tablePath;
 
-    public TableImpl(Path logPath, Path dataPath) {
-        this.logPath = logPath;
-        this.dataPath = dataPath;
-        this.snapshotManager = new SnapshotManager();
+    public TableImpl(String tablePath) {
```

Comment:
Why change the input from a `Path` to a String?

Reply:
Pretty sure we plan to remove `Path` completely (cc @vkorukanti) so wanted to avoid using it in a new public API